config/credentials.yml.enc

```diff
@@ -1 +1 @@
-X5sExIW75pfjZGQlTsYqUJHADhme9Tr//iSvvlN3qpIiBb98oaBazZR5GZmQJfyBfQSmQSSn4qXT7AKRnLcawQQIgiFXxRorE5P2jNrbV4sMCfpkIZL4db1Xa0tBx4Oga4NPzfJZdgWN+rGkEn9JfVaFPM8MyOzUjezBQkx6AFTdMliEv2+KeaGxqzZgiz7PYwYPQGcVLbMkjeZ69d9iFpN7mMo3DGuuVzFhvjDn5b4INOmtc9gaGSHX6fv2xLY6RT66fK6iWC/SqIDlXlqHiIP3kqHZYTZvFXcLXiS5eTWfct9+vbDApMkj2RbbGabZJpiIQn+W2vxy/fl6r1Ia93hNCI06r2ECO6iCSseRjSxYqvBT8rJV6pjrF4RLcs/eonjUO3pmaT/d1NXG+kLNu/z8PO8DOi9gwql2bfRs/gTxV472DeTZgFJAV6m3ZHhfHJe++eyNouaPXiYLIjY=--55kbFZc4UPP7Eywq--H+M1F+4WCADZzcXkm/GkOw==
\ No newline at end of file
+DB/Aka+J65Dpu9tcX+iWm1wzwzyqwGHg5SXCWEaUMcyaHdphiA1JIKmvAF5I/BVkhlEACs1NFrmAda3XSq/YGYRZG/iT7QuoqASrn+vXUFR70LE9DEKpq+2KM4h+tT57HaG3IytipkYIcWg+ygwW1Yz9kZdMSbrJ7Jjmp+Bn+0bGG7KvFMDOlQ29K9TTrBqPv99jQhAW2IiKXqVAUUdPuvmM0PvRcRQc75VHPh07qmnEByQVixAMVJZm2aartqZpgJ5zzYkVDL8Wk+YjzIPEB8TPhe8rMhnHbZDOM6qBdhbHkgYUO2qr1+oT4Kh+ba32a+0tCILV6sqF3y8A+GfGfmevhgfEs0ZokdVeKXGNVy44TjcTpO57DuQ4qvaBwU8aRywb/sk/3LU/5QLp8fEwKamQMljpOSbAhTc2MqxtAMYXXJ0kG/WWxXYC9e2sUc2Xx81O3Gg0LmzQhxKW+mbXopLqBDcZHw==--8fzTmCf1QygJZHTs--rC5khi8tu5kkTdeyY7KvAA==
```

Comment:
@igbanam This change uses my master key which I should share on Slack or another secure platform